spouts/rss/feed.php

```diff
@@ -212,7 +212,7 @@ public function getId() {
      */
     public function getTitle() {
         if ($this->items !== false && $this->valid()) {
-            return @current($this->items)->get_title();
+            return htmlspecialchars_decode(@current($this->items)->get_title());
```

Comment:
You are right, the behaviour was the same. And apparently, Feedly and tt-rss do it too. For now we can probably leave it as is